proto/axelar/evm/v1beta1/query.proto

```diff
@@ -129,6 +129,13 @@ message ChainsResponse {
             "github.com/axelarnetwork/axelar-core/x/nexus/exported.ChainName" ];
 }
 
+message CommandRequest {
+  string chain = 1;
+  string command_id = 2;
+}
+
+message CommandResponse { QueryCommandResponse command = 1; }
```

Comment:
I don't like the wrapping here. Maybe duplicate the fields if we want to avoid compat for the other API?

Reply:
sure I can re-populate the fields of  `QueryCommandResponse` in  `CommandResponse`.

Question for my benefit: what's the benefit of doing that? My understanding was that elements of `PendingCommandsResponse`'s result and result of `CommandResponse` were supposed to ever represent the same struct (ie `QueryCommandResponse`).

Reply:
just that the proto name is outdated, and changing the name would breaking the API for axelarons, but i prefer not to add more dependencies on that outdated name :/ Maybe other folks have a different preference though.

Reply:
I'm a bit confused. Why do you want to keep `QueryCommandResponse` around at all? The new `CommandResponse` is completely replacing it, isn't it?

Reply:
The `pending-commands` query uses it internally, which i don't like, but I don't want to the break API if possible (one option is to deprecate the old type, and use a new field and axelarons can migrate to that)